### PR TITLE
(fix:timeline): update v-for reverse

### PR DESCRIPTION
**Pull request by xiaoeyuZZZ**
- [ ] Component Migration

Fix #
(fix:timeline): update v-for reverse

### CODE REVIEW

**Review by bastarder — COMMENTED** [review]
Need to modify

**Comment by xiaoeyuZZZ** [comment]
I will test it, I think upgrading should not castrate the function, the code of the big guy is worth learning, considering where I did not expect, this kind of rigorous thinking is our feature





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "element-plus/element-plus"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年8月26日(星期三) 下午4:32
收件人:&nbsp;"element-plus/element-plus"<element-plus@noreply.github.com&gt;;
抄送:&nbsp;"zl"<772219457@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [element-plus/element-plus] (fix:timeline): update v-for reverse (#201)





 
@bastarder commented on this pull request.
 
Need to modify

 
In packages/timeline/src/index.vue:
 &gt; @@ -27,12 +33,24 @@ export default defineComponent({       */        return () =&gt; { +      let slots = ctx.slots.default?.() || [] +      const reverse = props.reverse +      if (reverse) { +        if(slots.length <= 1) { // 使用数组渲染slots.default?.()只有一个元素，内容在children里面  
i don't think this code can work correctly,
 did you try
   <el-timeline&gt;       <el-timeline-item v-for"..."&gt;</el-timeline-item&gt;       <el-timeline-item&gt;1</el-timeline-item&gt;       <el-timeline-item v-for"..."&gt;1</el-timeline-item&gt;   </el-timeline&gt;  
this example only use v-for, but actually you must consider width template , v-if,
 if you want to support reverse here, you can see here, but i think reverse is not necessary

File: packages/timeline/src/index.vue [hunk 1]
```diff
@@ -27,12 +33,24 @@ export default defineComponent({
      */
 
     return () => {
+      let slots = ctx.slots.default?.() || []
+      const reverse = props.reverse
+      if (reverse) {
+        if(slots.length <= 1) { // 使用数组渲染slots.default?.()只有一个元素，内容在children里面
```
[the remark this answers is not in the document]
Comment (zouhangwithsweet) [reply packages/timeline/src/index.vue:39]: cool,we need more border cases

[the remark this answers is not in the document]
Comment (xiaoeyuZZZ) [reply packages/timeline/src/index.vue:39]: Big guy, you're right. I've been thinking too much

**Pull request closed without merging**